test/utils/utils.test.js

```diff
@@ -234,6 +234,15 @@ describe('Utils', () => {
       expect(newTabLink.href).to.equal('https://www.adobe.com/test');
     });
 
+    it('Sets up milo.deferredPromise', async () => {
+      window.milo = {};
```

Comment:
If you decide to move the `window.milo` declaration as previously suggested, this would no longer be needed. Instead, you could check for its existence after calling `setupDeferredPromise`